### PR TITLE
[feature](mtmv)(6)implement cancel method

### PR DESCRIPTION
**Pull request by zddr**
## Proposed changes

Issue Number: close #xxx

1.implement cancel task method
2.fix `show create table ` not display `comment`

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by zddr** [comment]
run buildall

**Comment by zddr** [comment]
run buildall

File: regression-test/data/mtmv_p0/test_build_mtmv.out [hunk 1]
```diff
@@ -1,4 +1,7 @@
 -- This file is automatically generated. You should know what you did if you want to edit this
+-- !show --
+multi_mv_test_create_mtmv	CREATE TABLE `multi_mv_test_create_mtmv` (\n  `aa` BIGINT NULL COMMENT 'aaa',\n  `bb` VARCHAR(20) NULL\n) ENGINE=MATERIALIZED_VIEW\nCOMMENT 'comment1'\nDISTRIBUTED BY RANDOM BUCKETS 10\nPROPERTIES (\n"replication_allocation" = "tag.location.default: 1",\n"min_load_replica_num" = "-1",\n"is_being_synced" = "false",\n"storage_format" = "V2",\n"light_schema_change" = "true",\n"disable_auto_compaction" = "false",\n"enable_single_replica_compaction" = "false",\n"group_commit_interval_ms" = "10000",\n"enable_duplicate_without_keys_by_default" = "true"\n);
```
Comment (morrySnow) [inline regression-test/data/mtmv_p0/test_build_mtmv.out:3]: i think we should return create materialized view ... when do `show create table` on MATERIALIZED_VIEW

**Comment by zddr** [comment]
run buildall

**Comment by zddr** [comment]
run buildall

**Review by morningman — APPROVED** [review]
LGTM

**Review by CalvinKirs — APPROVED** [review]

**Pull request merged**